package.json

```diff
@@ -103,6 +107,13 @@
     "mssql:test": "DB=mssql npm run plaintest",
     "mssql:destroy": "docker-compose -f scripts/mssql-docker-compose.yml stop",
     "postmssql:init": "node scripts/wait-for-mssql-server.js && npm run mssql:logs || (npm run mssql:logs;false)",
+    "prepublish": "npm run babel",
+    "pre_test": "npm run lint && npm run lint_types",
+    "bin_test": "cross-env KNEX_PATH=../knex.js KNEX=bin/cli.js jake -f test/jake/Jakefile",
+    "tape": "node test/tape/index.js | tap-spec",
```

Comment:
duplicate with test:tape

Reply:
how about that "prepublishOnly", when that is ran?

Reply:
npm is throwing deprecation warning about prepublish now. Now there are two separate hooks: `prepublishOnly` for publish and `prepare` for after install. prepublish should actually be removed, next major npm version won't support it.

Reply:
thanks, I'll remove prepublish already then. Its really unlikely that we would publish anything without running babel first, since its pretty much included in every hook there :)